libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/listener/OracleModeSqlParserListener.java

```diff
@@ -700,4 +700,62 @@ public void enterRollback_stmt(Rollback_stmtContext ctx) {
         this.dbObjectType = DBObjectType.OTHERS;
     }
 
+    @Override
+    public void enterSet_comment_stmt(OBParser.Set_comment_stmtContext ctx) {
```

Comment:
what about set_session statement?

Reply:
oracle does not support the set session statement

Reply:
have added